VisualPinball.Unity/VisualPinball.Unity/Game/CameraClipPlane.cs

```diff
@@ -0,0 +1,142 @@
+// Visual Pinball Engine
+// Copyright (C) 2021 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+using System; 
+using Unity.Mathematics;
+using UnityEngine;
+
+
+namespace VisualPinball.Unity
+{
+	[ExecuteAlways]
```

Comment:
This means the life cycle is applied in the editor as well, right? What's the editor function of this MonoBehavior? I see that below you're fetching the table bounds differently whether you're in play mode or not, but the camera itself is only active in play mode, no?

Reply:
The game view panel rendering is used in both modes so if the camera is moved it needs to update the clipping planes for that view.  The camera is always active.

Reply:
Got it, thanks!

No serious performance percussions I assume?

Reply:
Nah, it's a pretty cheap check.  Just polling for a location change and if it notices one, it updates otherwise it skips.